manager/src/test/scala/org/broadinstitute/transporter/db/DbClientSpec.scala

```diff
@@ -64,4 +66,52 @@ class DbClientSpec extends FlatSpec with ForAllTestContainer with Matchers {
     val client = new DbClient(testTransactor("nope"))
     client.checkReady.unsafeRunSync() shouldBe false
   }
+
```

Comment:
Is there an easy way to simulate a queue creation error in order to test that the db entry gets deleted properly?

Reply:
Yes, was just hoping to be lazy 😅 I'll beef up the test coverage